db/db_test.cc

```diff
@@ -1257,6 +1257,7 @@ TEST_F(DBTest, ApproximateSizesMemTable) {
   options.compression = kNoCompression;
   options.create_if_missing = true;
   DestroyAndReopen(options);
+  auto defaultColumnFamily = db_->DefaultColumnFamily();
```

Comment:
the naming convention is `default_column_family` for variables. We follow Google C++ Style, which provides this guideline: https://google.github.io/styleguide/cppguide.html#Variable_Names

Reply:
ok